airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -215,6 +215,12 @@ def stream_slices(
                     continue
                 yield customer
 
+class CreditNotes(IncrementalStripeStream):
+    cursor_field = "created"
```

Comment:
@archangelic This class should similar to this one: https://github.com/airbytehq/airbyte/blob/5707e477ad35d354a250748bd633d79022b79191/airbyte-integrations/connectors/source-stripe/source_stripe/streams.py#L240-L257
because `CreditNotes` doesn't support incremental.

Reply:
The difference is CreditNotes allow you to set the `limit` parameter so you can keep that in `request_params`.